sdc/tests/tests_perf/test_perf_df.py

```diff
@@ -109,21 +109,21 @@ def _test_binary_operations(self, pyfunc, name, total_data_length,
 
 #  (method_name, parametrs, total_data_length, call_expression)
 cases = [
-    ('count', '', [10 ** 7]),
-    ('drop', 'columns="f0"', [10 ** 8]),
-    ('max', '', [10 ** 7]),
-    ('mean', '', [10 ** 7]),
-    ('median', '', [10 ** 7]),
-    ('min', '', [10 ** 7]),
-    ('pct_change', '', [10 ** 7]),
-    ('prod', '', [10 ** 7]),
-    ('std', '', [10 ** 7]),
-    ('sum', '', [10 ** 7]),
-    ('var', '', [10 ** 7]),
+    TestCase(name='count', size=[10 ** 7]),
```

Comment:
I propose to import `TestCase` everywhere something like that:
from .generator import TestCase as TC